inc/dashboard/namespace.php

```diff
@@ -146,7 +145,7 @@ function render_page() {
 function get_plugin_version() : string {
 	// Only show version if this is embedded in the accelerate plugin.
 	if ( defined( 'Altis\\Accelerate\\VERSION' ) ) {
-		return Altis\Accelerate\VERSION;
+		return \Altis\Accelerate\VERSION;
 	}
```

Comment:
Why would the `return` be "`\Altis\Accelerate\VERSION;`" but the `if` statement is "`Altis\\Accelerate\\VERSION`"

Reply:
Should be `Altis\Accelerate\VERSION` i think, I missed that, good shout!

Reply:
In the if statement it's escaped using double backslashes as we're checking for the existence of a constant by name as a string. In that instance the `defined()` function isn't scoped to the current namespace. Adding the preceding backslash before the `return` statement is just to make absolutely certain we're reading from the root of the namespaces.

Reply:
Thanks @roborourke